apps/bam.iobio/src/App.tsx

```diff
@@ -131,15 +131,16 @@ function App() {
 						<iobio-data-broker url="https://s3.amazonaws.com/iobio/NA12878/NA12878.autsome.bam" />
 
 						<div className="row iobio-container">
-							{mappedReads ? <iobio-percent-box percent-key="mapped_reads" total-key="total_reads" /> : <></>}
-							{forwardStrands ? <iobio-percent-box percent-key="forward_strands" total-key="total_reads" /> : <></>}
-							{properPairs ? <iobio-percent-box percent-key="proper_pairs" total-key="total_reads" /> : <></>}
-							{singletons ? <iobio-percent-box percent-key="singletons" total-key="total_reads" /> : <></>}
-							{bothMatesMapped ? <iobio-percent-box percent-key="both_mates_mapped" total-key="total_reads" /> : <></>}
-							{duplicates ? <iobio-percent-box percent-key="duplicates" total-key="total_reads" /> : <></>}
+							{mappedReads && <iobio-percent-box percent-key="mapped_reads" total-key="total_reads" />}
+							{forwardStrands && <iobio-percent-box percent-key="forward_strands" total-key="total_reads" />}
+							{properPairs && <iobio-percent-box percent-key="proper_pairs" total-key="total_reads" />}
+							{singletons && <iobio-percent-box percent-key="singletons" total-key="total_reads" />}
+							{bothMatesMapped && <iobio-percent-box percent-key="both_mates_mapped" total-key="total_reads" />}
+							{duplicates && <iobio-percent-box percent-key="duplicates" total-key="total_reads" />}
 						</div>
 						<div className="row iobio-histo-container">
-							{histogram ? <iobio-histogram broker-key="coverage_hist" /> : <></>}
+							{histogram && <iobio-histogram broker-key="coverage_hist" />}
+							{histogram && <Histogram broker-key="coverage_hist" />}
```

Comment:
This implementation is only the starter code for the React component version of Histogram
It only shows that TSX imports are working